packages/legacy/core/App/components/misc/CredentialCard.tsx

```diff
@@ -3,7 +3,7 @@ import { Attribute, BrandingOverlayType, Predicate } from '@hyperledger/aries-oc
 import React from 'react'
 import { ViewStyle } from 'react-native'
 
-import { TOKENS, useContainer } from '../../container-api'
+import { TOKENS, useServices } from '../../container-api'
```

Comment:
Curious why this was renamed? To match Credo-ts?

Reply:
useServices is a different hook, is allows getting multiple things from the config with one call, clecio added it in